.github/COMMITTING.md

```diff
@@ -0,0 +1,39 @@
+# Committing to Nemo
+
```

Comment:
How about structuring it as follows? 
* For contributor
   * ... 
* For committer 
   * ...

Reply:
same here